src/main.cc

```diff
@@ -268,10 +270,51 @@ static void print_help(const char *prog_name) {
          "   -i COUNT                  number of times to update " PACKAGE_NAME
          " (and quit)\n"
          "   -p, --pause=SECS          pause for SECS seconds at startup "
-         "before doing anything\n",
+         "before doing anything\n"
+         "   -U, --unique              only one conky process can be created\n",
          prog_name);
 }
 
+#if defined(__linux__)
+// NOTE: Only works on systems where there is a /proc/[pid]/stat file.
+static bool is_conky_already_running() {
```

Comment:
Minor nit pick: can you relocate this function to `src/linux.cc`?

Reply:
Sure.